Add agent configuration guide (#451)
xmtp/docs-xmtp-org: docs/pages/fund-agents-apps/configure-agent.mdx

# Configure your agent

Your agent needs to pay for the messages it sends on the XMTP network. To get started, you'll need.

1. An agent built using our Node or Agent SDK
2. A payer wallet that has been [funded through the XMTP Funding Portal](./fund-your-app.mdx)

> ## Setup your Signer

Reviewer: ```suggestion
## Setup your Payer Wallet
```

> When you create a client, you already specify a `Signer` that is used to link your XMTP identity to a wallet. You can choose to use the same `Signer` to pay for messages, or create a new one purely for this purpose.

Reviewer: ```suggestion
When you create a client, you already specify a `Signer` that is used to link your XMTP identity to a wallet. You can choose to use the same `Signer` as your payer wallet, or create a new one purely for this purpose. In either case, it must be the same wallet you [funded through the XMTP Funding Portal](./fund-your-app.mdx)
```

> This Signer must be of type `eoa`. The `messagePayer` may not be a smart contract wallet, since the signatures need to be able to be verified offchain.

Reviewer: ```suggestion
This Signer must be of type `eoa`. The `messagePayer` may not be a smart contract wallet, since the signatures need to be able to be verified offchain.

:::info

You can fund your payer wallet from a smart contract wallet [using the XMTP Funding Portal](./fund-your-app.mdx).

:::
